simple_network_sim/loaders.py

```diff
@@ -307,7 +307,7 @@ class MixingMatrix:
     age-ranges.
     
     Examples:    
-    `mm = MixingMatrix("sample_input_file/sample_20200327_comix_social_contacts.sampleCSV")`
+    `mm = MixingMatrix("sample_input_file/sample_20200327_comix_social_contacts.sampleCsv")`
```

Comment:
`.sampleCsv` -> `.csv`?

Should this point to the "simplified" file (sample_input_files/simplified_age_infection_matrix_from_comix.csv)? Would it run with either?

Reply:
This class will run with either, but you need the simplified one when you run the simulation (otherwise the age groups will not match).